.github/workflows/plyer_deploy.yml

```diff
@@ -0,0 +1,30 @@
+name: Deploy to PyPI
+
+on:
+  release:
+    types: [created]
```

Comment:
Are you sure that only tags are created and not e.g. branches? It may be better to run this simply `on: create`, and then have a check that it is a tag explicitly: `if: startsWith(github.ref, 'refs/tags/')` in the job root.

Kinda like at https://github.com/kivy-garden/graph/blob/master/.github/workflows/pythonapp.yml#L51. The workflow then is someone creates a tag locally or on gh. The CI runs and generates the sdist, the wheel and uploads to pypi and creates a draft release on github's release page and adds the files. Then we just need to publish the release draft and everything is set. This seems to work quite well.

Reply:
As far as I know, the `on: release types: [created]` will run the job when the release is created in GitHub ([excluding the draft releases](https://help.github.com/en/actions/reference/events-that-trigger-workflows#release-event-release)). Using [this method](https://help.github.com/en/github/administering-a-repository/managing-releases-in-a-repository), a tag is required. I don't see in documentation a place where it says it creates a branch or something that is not a tag.

Maybe you do not prefer this method. If I understood, you want plyer_deploy to run just with the creation of a tag, creating an artifact and a draft release, don't you? If so, I can work on it.

Reply:
>Maybe you do not prefer this method. If I understood, you want plyer_deploy to run just with the creation of a tag, creating an artifact and a draft release, don't you? If so, I can work on it.

Yes, I'd prefer running it with a tag as that is in line with our overall workflow. Thanks

Reply:
I just added those. Can you review the changes please?